packages/firebase_crashlytics/firebase_crashlytics_platform_interface/pubspec.yaml

```
@@ -8,16 +8,16 @@ environment:
   flutter: ">=1.12.13+hotfix.5 <2.0.0"
 
 dependencies:
+  collection: ^1.14.3
+  firebase_core: ^0.7.0-nullsafety.0
   flutter:
     sdk: flutter
   meta: ^1.0.5
```

This needs bumping to use a nullsafety version

```suggestion
  meta: ^1.3.0-nullsafety.0
```